docs/pages/setup/admin/adding-nodes.mdx

```diff
@@ -0,0 +1,260 @@
+---
+title: Adding Nodes to the cluster
+description: Adding Nodes to the cluster
+---
+
+This guide explains how to label nodes with Open Source, Enterprise Teleport,
```

Comment:
Looks like a copy paste from the labeling nodes guide. Don't we mean to say "this guide explains how to add nodes to a teleport cluster?"